build_library/catalyst_toolchains.sh

```diff
@@ -16,10 +16,13 @@ configure_target_root() {
     local cross_chost=$(get_board_chost "$1")
     local profile=$(get_board_profile "${board}")
 
+    # Get the host CBUILD.
+    local cbuild="$(portageq envvar CBUILD)"
+
     CHOST="${cross_chost}" \
         ROOT="/build/${board}" \
         SYSROOT="/usr/${cross_chost}" \
-        _configure_sysroot "${profile}"
+        _configure_sysroot ${cbuild} "${profile}"
```

Comment:
Hi,

I pushed out a new version that uses an environment variable, and is also rebased to the latest.

With the environment variable, we need to set CBUILD before CHOST for $(portageq envvar CBUILD) to still pick up the host tuple.  Should we put a comment in about this?

Also, we need to modify _configure_sysroot regardless to write the CBUILD to the make.conf file.  See the new patch.